README.md

```diff
@@ -1,123 +1,90 @@
-feedback
-========
+# Feedback
 
-This app handles user feedback related things.
+This is a public facing Ruby on Rails application that collects feedback from users in multiple ways (which are detailed below). This feedback information is then used to raise tickets in Zendesk.
 
-### Testing
+## Contact Page on GOV.UK
+![Contact Page on GOV.UK](/docs/screenshots/contact_page.png?raw=true "Contact Page on GOV.UK")
 
-To run unit tests, execute the following:
+## Contact Form on GOV.UK
+![Contact Form on GOV.UK](/docs/screenshots/contact_form.png?raw=true "Contact Form on GOV.UK")
 
-    bundle exec rake
+### Technical documentation
 
-### Testing with a mock `signon` strategy
+The app handles user feedback in the following ways:
 
-Launch using `bowl` from the `development` directory:
+1. Renders a contact page on GOV.UK: `https://www.gov.uk/contact/govuk`
+2. Collects feedback from other contact forms. Pages using the `static` app will have a link at the bottom of the page asking: `Is there anything wrong with this page?`. Clicking this link will show you a form. The data from the form will be submitted to the `feedback` app.
+3. Raises tickets based on feedback to Zendesk. The app does this by submitting data to the `support` app.
```

Comment:
Some tickets are submitted to the support app, which I think means they are sent to zendesk, but the majority of tickets are not, they're sent to the support-api app which might send them to zendesk, but equally might store them in a database.  It all depends on the type of feedback.  

It's out of the scope of this PR but it's beginning to feel to me like we might want to move the bits of support that feedback talks to out and into support-api and then we can close the communications between feedback and support, which *might* make things simpler.

Reply:
Shall I open a card about that?

Reply:
We talked about this face-to-face, but for those following along - there's a meeting to talk about feedback in general coming up soon which may involve changing feedback entirely (we don't know yet).  Once we've had that meeting we'll decide if we should re-architect the feedback/support/support-api interactions.